src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -96,4 +107,24 @@ public String getName(String userId, DatastoreService datastore) {
         String name = (String) userEntity.getProperty("name");
         return name;
     }
+
+    public boolean isTutor(String userId, DatastoreService datastore) {
+        Filter filter = new FilterPredicate("userId", FilterOperator.EQUAL, userId);
+        Query query = new Query("Tutor").setFilter(filter);
+        PreparedQuery results = datastore.prepare(query);
+
+        Entity tutorEntity = results.asSingleEntity();
+
+        return tutorEntity != null;
```

Comment:
I think there is a "role" property in the User entity that will tell you if the user is a tutor or a student.

Reply:
Oh cool! I'll look into it

Reply:
Unless we're accounting for people that are both a student and a tutor? In that case, should we make the "role" property equal to something like "both"?

Reply:
Yep,once I implement the possibility of registering as both student and tutor I'll make sure to add the distinction here and add the tab to go from the tutor's view to the student's view.